### ISSUE TITLE
Accessibility Testing for Veteran Facing Forms - Form 21-4138

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#102850]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

Root page used for testing
- https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/

### Who completed the use of color and color contrast test?

@jeana-adhoc 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

No issues

### Who completed the axe scans?

@jeana-adhoc 

### axe checks

- [ ] Each page has been scanned using axe (results shared below)
- [X] axe is integrated into your end-to-end testing

### axe DevTools scan results

| Axe Results   | Page           |
| ------------- | -------------- |
| Content not in a landmark (true for all pages of this form) | [Introduction](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/introduction) |
| Same as intro | [Statement Type](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/statement-type) |
| Same as intro | [CST](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/claim-status-tool) |
| Same as intro | [Decision review](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/decision-review) |
| Same as intro | [Lay Witness statement](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/lay-witness-statement) | 
| Same as intro | [Priority processing](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/priority-processing) |
| Same as intro | [Personal Records](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/personal-records-request) |
| Same as intro | [Name & DOB](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/personal-information) |
| Same as intro | [Identification information](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/identification-information) |
| Same as intro | [Mailing address](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/mailing-address) |
| Same as intro | [Contact info](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/contact-information) |
|  Same as intro | [Statement](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/statement)  |
| Missing h1 & Same as intro | [Review](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/review-and-submit) |
| Missing H1 & same as intro | [Confirmation page](https://staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/confirmation) |

### How did axe testing go?

Axe helped reveal a couple of issues.

1. For some reason the "Skip to content" link is being marked as not being in a landmark area.  I think this is related to the minimal form layout. We see the same issue in the [mock minimal form layout](https://staging.va.gov/mock-form-minimal-header/introduction) (Nav to name & dob page) but not in the default/current [mock form layout](https://staging.va.gov/mock-form-patterns/introduction)  (Nav to name & dob page). [A ticket has been filed](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2037).
2. The review page is still missing an H1. [A ticket for this was already filed](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2023)
3. The confirmation page is also missing an `H1`. The confirmation page layout should be similar to that of the intro page, in that the `H1` is the plain language name of the form.  [Ticket has been filed](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2067)

### Who completed the content zoom and reflow test?

@jeana-adhoc 

### Content zoom and reflow checks

- [X] All page elements are readable and usable at 200% zoom
- [X] All page elements are readable and usable at 300% zoom
- [X] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?
Most of the form did just fine under 2, 3, 400% zoom. There were only issues (and a minor one) on pages that used the updated primary action arrow. When zoomed in at larger percentages and there's a lot of text in the updated "primary action link/arrow", the point of the arrow does not expand with the line height.

<img width="450" alt="Image" src="https://github.com/user-attachments/assets/6c08a02b-94f1-4773-98a2-dd27b18be669" />

[A ticket was previously filed for this issue](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1449) However, we've not filed this with the design system yet, and there hasn't been any DST incorporation of the updated styles yet. ([See this experimental design ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1459))

### Who completed the keyboard navigation test?

@jeana-adhoc 

### Keyboard navigation checks

- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [X] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [X] All elements under focus have a visible focus indicator
- [X] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

No issues

### Do you have any other results to share?

1. When clicking the "submit this application" button on the review page, the button then revealed the back and submit button while the page tried to submit. This seemed odd, and could cause user error if they think they can click on the back button. You can see an example in this video. 
https://github.com/user-attachments/assets/2c498a4a-97c5-4d31-a402-4b396377a597

[Bug ticket filed](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2038)

2. If I navigate to one of the secondary forms and load its introduction page, and then use the back button in the browser, the standard/desktop breadcrumbs end up displaying on the page as well as the "back to the previous page" breadcrumb. You can see an example in this video.
https://github.com/user-attachments/assets/b71c2d82-a927-4e9b-b706-1f4d360fc4e8

[Bug ticket filed](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2039)

**Comment by jeana-adhoc** [comment]
**Related tickets filed** 

- [All page content should be in landmark areas](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2037)
- [Add H1 to review page in minimal form flow](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2023)
- [Text wrapping on new action link](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/1449)
- [Phantom back button](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2038)
- [Breadcrumbs](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2039)

At the time I did this testing, the confirmation page was not loading. This will be tested as soon as possible.

**Comment by tbaker1026** [comment]
[Confirmation page should have an H1](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/issues/2067)
